lib/compat/wordpress-6.5/class-wp-navigation-block-renderer.php

```diff
@@ -0,0 +1,611 @@
+<?php
+/**
+ * WP_Navigation_Block_Renderer class
+ *
+ * @package gutenberg
+ * @since 6.5.0
+ */
+
+if ( class_exists( 'WP_Navigation_Block_Renderer' ) ) {
+	return;
+}
+
+/**
+ * Helper functions used to render the navigation block.
+ */
+class WP_Navigation_Block_Renderer {
+
+	private static $list_item_nav_blocks = array(
```

Comment:
I don't think this var is well named. What does it mean?

PS: I know you didn't come up with the name but can we improve it?

Reply:
renamed to `nav_blocks_wrapped_in_list_item` in aceed2e815